submissions/tests/test_api.py

```diff
@@ -169,11 +169,11 @@ def test_get_submission(self):
             api.get_submission("deadbeef-1234-5678-9100-1234deadbeef")
 
     @mock.patch.object(Submission.objects, 'get')
-    @raises(api.SubmissionInternalError)
     def test_get_submission_deep_error(self, mock_get):
         # Test deep explosions are wrapped
-        mock_get.side_effect = DatabaseError("Kaboom!")
-        api.get_submission("000000000000000")
+        with raises(api.SubmissionInternalError):
```

Comment:
we should avoid dependency on pytest instead use `TestCase`'s `assertRaises` method 
`with self.assertRaises(api.SubmissionInternalError):`

Reply:
@ziafazal changes have been made. please review it again.